cloud/aws/deploytf/service.go

```diff
@@ -54,7 +53,6 @@ func (a *NitricAwsTerraformProvider) Service(stack cdktf.TerraformStack, name st
 		// "NITRIC_AWS_RESOURCE_RESOLVER": jsii.String("tagging"),
 	}
 
-	// TODO: Only apply to requesting services
```

Comment:
Still relevant as databases currently don't have 'requesting service' metadata, all lambdas get access to the database currently